apps/src/templates/progress/BubbleBadge.jsx

```diff
@@ -10,44 +10,76 @@ import FontAwesome from '../FontAwesome';
 const badgeTypes = ['assessment', 'keepWorking'];
 
 function BubbleBadge({type, isDiamond}) {
```

Comment:
this is probably mostly because the bottom of the file was collapsed, but it took me awhile to figure out how this component was being used since it's not exported. generally we seem to be pretty consistent about putting `export default` on this line when it's not a connected or Radium component, so imo it would be a little clearer to do that here